src/oas3/transformers/callbacks.ts

```diff
@@ -19,11 +20,14 @@ export const translateToCallbacks: Oas3TranslateFunction<[callbacks: unknown], I
             paths: { [path]: { [method]: op } },
           };
 
+          const ctx = createContext(document);
+          ctx.context = 'callback';
```

Comment:
you need to copy over ids.
```suggestion
          Object.assign(ctx.ids, this.ids);
```

Reply:
I added `Object.assign(ctx.ids, this.ids);` but I also retained `ctx.context = 'callback';`

Reply:
I went ahead and updated to exactly you suggestion in the meantime